docs/topics/downloader-middleware.rst

```diff
@@ -429,6 +429,40 @@ In order to use this policy, set:
 * :setting:`HTTPCACHE_POLICY` to ``scrapy.extensions.httpcache.RFC2616Policy``
 
 
+.. _httpcache-policy-custom:
+
+Writing your own policy
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can implement a cache policy by creating a class that
+defines the methods described below.
+
+.. module:: scrapy.extensions.httpcache
+
+.. class:: CachePolicy
+
+    .. method:: should_cache_request(request)
+
+      Return True if the :class:`~scrapy.http.Request` is cacheable otherwise False
+
+    .. method:: should_cache_response(response, request)
+
+      Return True if :class:`~scrapy.http.Response` should be cached otherwise False
+
+    .. method:: is_cached_response_fresh(cachedresponse, request)
+
+      Return True if cached :class:`~scrapy.http.Response` has expired otherwise False
```

Comment:
This seems contradictory with the name of the method. Could you please check that `True` is not returned when the specified cached response has *not* expired?